Patch-Source: https://github.com/php/php-src/commit/046b36bcf8c062375c9f5e2a763d6144c2a484b4
From 046b36bcf8c062375c9f5e2a763d6144c2a484b4 Mon Sep 17 00:00:00 2001
From: Nikita Popov <nikita.ppv@gmail.com>
Date: Wed, 4 Aug 2021 11:55:47 +0200
Subject: [PATCH] Use different cipher in openssl_seal() test

RC4 is insecure and not supported in newer versions.
---
 ext/openssl/tests/openssl_seal_basic.phpt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/ext/openssl/tests/openssl_seal_basic.phpt b/ext/openssl/tests/openssl_seal_basic.phpt
index 12c2824ee1b3..2d9d2cdd263c 100644
--- a/ext/openssl/tests/openssl_seal_basic.phpt
+++ b/ext/openssl/tests/openssl_seal_basic.phpt
@@ -9,7 +9,7 @@ $a = 1;
 $b = array(1);
 $c = array(1);
 $d = array(1);
-$method = "RC4";
+$method = "AES-128-ECB";
 
 var_dump(openssl_seal($a, $b, $c, $d, $method));
 
@@ -41,8 +41,8 @@ var_dump(openssl_seal($data, $sealed, $ekeys, array($wrong), $method));
 Warning: openssl_seal(): Not a public key (1th member of pubkeys) in %s on line %d
 bool(false)
 openssl_seal(): Argument #4 ($public_key) cannot be empty
-int(19)
-int(19)
+int(32)
+int(32)
 
 Warning: openssl_seal(): Not a public key (2th member of pubkeys) in %s on line %d
 bool(false)
